test/Microsoft.ML.Core.Tests/UnitTests/TestHosts.cs

```diff
@@ -83,7 +83,7 @@ public void LogEventProcessesMessages()
             env.Log += (sender, e) => messages.Add(e.Message);
 
             // create a dummy text reader to trigger log messages
-            env.Data.CreateTextReader(
+            env.Data.CreateTextLoader(
```

Comment:
We should get this change in soon since the build is currently broken. If we can't check this in right away, let's get a separate PR out with just fixing the build.

/cc @najeeb-kazmi